tools/ampir/ampir_Rscript

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env Rscript
```

Comment:
Philosphical reason. The user wanting to fix a converted module will need to get the script working first so it is supplied as a convenience - no need to copypasta - just setup a conda env and get it working on the command line - then it can be fixed in my nf-core module fork permanently so the next build run will work correctly.
Will permanently remove it if you think it is a Bad Thing to include although I disagree - please let me know.